src/overlays/actors/ovl_Dm_Char08/z_dm_char08.c

```diff
@@ -15,7 +16,41 @@ void DmChar08_Destroy(Actor* thisx, PlayState* play);
 void DmChar08_Update(Actor* thisx, PlayState* play);
 void DmChar08_Draw(Actor* thisx, PlayState* play);
 
-#if 0
+void func_80AAFAC4(DmChar08* this, PlayState* play);
+void DmChar08_WaitForSong(DmChar08* this, PlayState* play);
+void func_80AAF8F4(DmChar08* this, PlayState* play);
+void func_80AAFAE4(DmChar08* this, PlayState* play);
+void func_80AAFE78(DmChar08* this, PlayState* play);
+void func_80AAFA18(DmChar08* this, PlayState* play);
+void DmChar08_SetupAppearCs(DmChar08* this, PlayState* play);
+void func_80AAF884(DmChar08* this, PlayState* play);
+void func_80AAFB04(DmChar08* this, PlayState* play);
+void func_80AAFB94(DmChar08* this, PlayState* play);
+void func_80AAFE88(DmChar08* this, PlayState* play);
+void func_80AB023C(DmChar08* this, PlayState* play);
+void func_80AB01E8(DmChar08* this, PlayState* play);
+void DmChar08_SpawnBubbles(DmChar08* this, PlayState* play);
+void func_80AAFCCC(DmChar08* this, PlayState* play);
+
+typedef enum {
+    EYEMODE_0,
+    EYEMODE_1,
+    EYEMODE_2,
+    EYEMODE_3,
+    EYEMODE_4,
+    EYEMODE_5,
+} EyeMode;
+
+typedef enum {
+    ANIM_0,
+    ANIM_1,
+    ANIM_2,
+    ANIM_3,
+    ANIM_4,
+    ANIM_5,
+    ANIM_6,
+} ANIM_MODE;
```

Comment:
These should still have a unique prefix, and the type names should too, and be PascalCase.